fcrepo-audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/AuditSparqlProcessor.java

```diff
@@ -66,8 +65,8 @@
     public void process(final Exchange exchange) throws Exception {
```

Comment:
I would have expected updates to the following tests to be included in this PR:
`AuditSparqlIT` and `ProcessorTest`